scripts/package/projectsToPack.ps1

```diff
@@ -8,5 +8,7 @@ $ProjectsToPack = @(
     "Microsoft.DotNet.ProjectModel.Workspaces",
     "Microsoft.DotNet.InternalAbstractions",
     "Microsoft.Extensions.DependencyModel",
-    "Microsoft.Extensions.Testing.Abstractions"
+    "Microsoft.Extensions.Testing.Abstractions",
```

Comment:
(Really minor) Would it make sense to keep this list alphabetized?  That way we can quickly scan for any missed packages by looking at the folders under "src".